dxr/plugins/clang/dxr-index.cpp

```diff
@@ -1115,29 +1135,31 @@ class IndexConsumer : public ASTConsumer,
       StringRef searchPath,
       StringRef relativePath,
       const Module *imported) {
-    PresumedLoc presumedHashLoc;
-    FileInfo *target, *source;
-    SourceLocation targetBegin, targetEnd;
+    PresumedLoc presumedHashLoc = sm.getPresumedLoc(hashLoc);
```

Comment:
(I now see that some of this was likely due to switching to returning refs from getFileInfo().